ext/reflection/php_reflection.stub.php

```diff
@@ -826,3 +826,26 @@ public function getCallable(): callable {}
 
     public function getTrace(int $options = DEBUG_BACKTRACE_PROVIDE_OBJECT): array {}
 }
+
+/**
+ * @strict-properties
+ * @not-serializable
+ */
+final class ReflectionConstant implements Reflector
+{
+    public string $name;
+
+    public function __construct(string $name) {}
+
+    public function getName(): string {}
+
+    public function getNamespaceName(): string {}
+
+    public function getShortName(): string {}
```

Comment:
I would have imagined that `getName()` just returns the name of the constant, not the FQN

Reply:
As discussed privately :slightly_smiling_face: